code/modules/mob/living/carbon/human/species_types/kepori.dm

```diff
@@ -31,6 +31,20 @@
 	no_equip = list(ITEM_SLOT_BACK)
 	mutanttongue = /obj/item/organ/tongue/kepori
 	species_language_holder = /datum/language_holder/kepori
+	/// For storing our tackler datum so we can remove it after
+	var/datum/component/tackler
+	/// See: [/datum/component/tackler/var/stamina_cost]
+	var/tackle_stam_cost = 10
+	/// See: [/datum/component/tackler/var/base_knockdown]
+	var/base_knockdown = 0.2 SECONDS
+	/// See: [/datum/component/tackler/var/range]
+	var/tackle_range = 8
+	/// See: [/datum/component/tackler/var/min_distance]
+	var/min_distance = 1
+	/// See: [/datum/component/tackler/var/speed]
+	var/tackle_speed = 2
+	/// See: [/datum/component/tackler/var/skill_mod]
+	var/skill_mod = 2
```

Comment:
aghhh this is nasty. But it's.. probably fine?

Reply:
I mean, the only thing that *has* to be there is the component var so I can just make the rest defined constants if that's preferable.